library/src/main/java/com/pengrad/telegrambot/TelegramBot.java

```diff
@@ -104,16 +105,22 @@ public Builder apiUrl(String apiUrl) {
             return this;
         }
 
+        public Builder fileApiUrl(String fileApiUrl) {
+            this.fileApiUrl = fileApiUrl;
+            return this;
+        }
+
         public Builder updateListenerSleep(long millis) {
             updatesHandler = new UpdatesHandler(millis);
             return this;
         }
 
         public TelegramBot build() {
-            if (okHttpClient != null || apiUrl != null) {
+            if (okHttpClient != null || apiUrl != null || fileApiUrl != null) {
```

Comment:
fileApiUrl check should be in separate if block.
okHttpClient and apiUrl relate to `api` object, fileApiUrl to `fileApi`